### ISSUE TITLE
SyntaxWarning: "is not" with a literal. Did you mean "!="?

### ISSUE THREAD

**Issue by sm4rk0** [go-macaroon-bakery/py-macaroon-bakery#81]
I've just updated the Ubuntu (20.04 - devel branch) package and saw this:
```
Preparing to unpack .../python3-macaroonbakery_1.2.3-3_all.deb ...
Unpacking python3-macaroonbakery (1.2.3-3) over (1.2.3-1) ...
Setting up python3-macaroonbakery (1.2.3-3) ...
/usr/lib/python3/dist-packages/macaroonbakery/bakery/_macaroon.py:119: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if cav.condition is not '':
```

**Comment by fransvanberckel** [comment]
While setting up python3-macaroonbakery 1.3.0-1, it still there.

**Comment by frankban** [comment]
The fix has been released as 1.3.1.
